docs/introduction/introduction.md

```diff
@@ -33,7 +54,7 @@ _Easy installation for use with prepared Docker images and Helm charts._ Metron
 - Security teams can easily create a rule-based alert from a single data source, or they can create advanced correlation rules that combine various data sources.
 - Siembol UI supports translating Sigma rule specification (generic and open signature format for SIEM alerting [https://github.com/SigmaHQ/sigma](https://github.com/SigmaHQ/sigma)) into the Siembol alerting rule.
 
-### Integration with Other Systems – Siembol Response
+### Integration with Other Systems-Siembol Response
```

Comment:
Revert to including the spaces. The spaces are intentional since it changes it to a single hyphenated word rather than two separate words as intended.

Reply:
So sorry about that! Just changed it!